checks-superstaq/checks_superstaq/pytest_.py

```diff
@@ -82,11 +87,14 @@ def run(
     if not files:
         return 0
 
+    if not parsed_args.single_core and len(files) != 1:
+        args_to_pass += ["-n=auto"]
```

Comment:
this should be put before the other `args_to_pass` so command line arguments will take precedence (for example currently i can pass `./checks/pytest_.py -n0` to override `-n=auto` and disable threading)

Reply:
you can disable threading with the new `--single_core` command.

Also are you referring to `parser.parse_known_intermixed_args` for the other `args_to_pass`?

Reply:
sure but currently i can also pass in `-n2`/`-n5`/etc :) and also it's nice to have the same arguments work whether i'm using `./checks/pytest_.py` or `pytest` directly. This would work if we just change this to e.g.
```suggestion
        args_to_pass = ["-n=auto", *args_to_pass]
```

(im just referring to the other values in `args_to_pass` - which contains any arguments i pass in to `./checks/pytest_.py` that aren't recognized by the parser)

Reply:
Ah, I see. I'll change this